### ISSUE TITLE
Unable to delete a report?

### ISSUE THREAD

**Issue by angieherrera** [masugadesign/lab-reports-craft-cms#5]
Unless I missed something, I don't see a way to delete a report?

I deleted the template for the report in the config folder but the report still shows in the CP. The delete option is grayed out in the CP if I try that way.

**Comment by amici-infotech** [comment]
I am having this issue too. Can't delete the report.

![image](https://github.com/masugadesign/lab-reports-craft-cms/assets/20450552/50cd71ca-0699-4b5f-b64a-81cc14650ead)

**Comment by amici-infotech** [comment]
After debug I found we need to add 2 functions into element class. I've added this into `ConfiguredReport.php` file.

```
// At top
use craft\elements\User;

// Under class
public function canView(User $user): bool
{
    return true;
}

public function canDelete(User $user): bool
{
    return true;
}
```

**Comment by benjaminkohl** [comment]
Thank you for bringing this to my attention. I'll be sure to address permissions issues in a future release. I don't think Craft always enforced these unless they were explicitly defined on the element. I never had any intention of getting that granular with the plugin permissions.

**Comment by benjaminkohl** [comment]
I just released Lab Reports 2.0.3 which will address the permissions issues that I _think_ may have come about in Craft 4.3 (did some digging and comparing of Craft 4 releases).
